packages/strapi/lib/core/app-configuration/env-helper.js

```diff
@@ -2,42 +2,58 @@
 
 const _ = require('lodash');
 
-function env(key, defaultValue = undefined) {
+function env(key, defaultValue) {
   return _.has(process.env, key) ? process.env[key] : defaultValue;
 }
 
 const utils = {
-  int(...args) {
-    const value = env(...args);
-    return typeof value === 'undefined' ? value : parseInt(value, 10);
+  int(key, defaultValue) {
+    if (!_.has(process.env, key)) {
+      return defaultValue;
+    }
+
+    const value = process.env[key];
+    return parseInt(value, 10);
   },
 
-  bool(...args) {
-    const value = env(...args);
-    return typeof value === 'undefined' ? value : value === 'true';
+  float(key, defaultValue) {
+    if (!_.has(process.env, key)) {
+      return defaultValue;
+    }
+
+    const value = process.env[key];
+    return parseFloat(value);
   },
 
-  float(...args) {
-    const value = env(...args);
-    return typeof value === 'undefined' ? value : parseFloat(value);
+  bool(key, defaultValue) {
+    if (!_.has(process.env, key)) {
+      return defaultValue;
+    }
+
+    const value = process.env[key];
+    return ['true', true].includes(value);
```

Comment:
`true` is not necessary now since `process.env[key]` is always a string. value === 'true' would work.